advanced/Scripts/version.sh

```diff
@@ -91,11 +91,19 @@ getRemoteVersion(){
     #If the above file exists, then we can read from that. Prevents overuse of GitHub API
     if [[ -f "$cachedVersions" ]]; then
         IFS=' ' read -r -a arrCache < "$cachedVersions"
-        case $daemon in
-          "pi-hole"   )  echo "${arrCache[0]}";;
-          "AdminLTE"  )  echo "${arrCache[1]}";;
-          "FTL"       )  echo "${arrCache[2]}";;
-        esac
+
+        if [[ "${INSTALL_WEB_INTERFACE}" == true ]]; then
+            case $daemon in
+                "pi-hole"   )  echo "${arrCache[0]}";;
+                "AdminLTE"  )  echo "${arrCache[1]}";;
```

Comment:
How about something like this to reduce a duplication of the case statement below?

```suggestion
                "AdminLTE"  ) [[ "${INSTALL_WEB_INTERFACE}" == true ]] && echo "${arrCache[1]}";;
```